sources/packages/api/src/route-controllers/institution/institution.institutions.controller.ts

```diff
@@ -35,6 +35,7 @@ export class InstitutionInstitutionsController extends BaseController {
    * Get institution details of given institution.
    * @returns InstitutionDetailDTO
    */
+  @IsInstitutionAdmin()
   @Get()
   async getInstitutionDetail(
     @UserToken() token: IInstitutionUserToken,
```

Comment:
is this `InstitutionDetailDTO` renaming part of future PR

Reply:
No I will adjust the comments.

Reply:
Yes that is not in scope of this PR.